books.js

```diff
@@ -40,12 +40,14 @@ function getBooks() {
   collection.forEach((book, index) => {
     innerhtml += `
     <div>
-        <div>${book.title} </div>
-        <div> ${book.author} </div>
+ 
+     <p>  "${book.title}" by 
+         ${book.author} 
+     </p>
         <button id="remove-btn${index}">Remove</button>
             </div>
-            <hr>
-        `;
+            
+    `;
   });
 
   booklist.innerHTML = innerhtml;
```

Comment:
- [ ] Hi @ismailmunyentwari9, I think we still need to refactor the project. Based on the requirement of this project, arrays and objects can be used but it has to be inside the class. At the very top of this file, you define `collection = []` . This is an array which you pushing and removing a book from in the `addBook` and `removeBook` methods which are not still defined inside a class.


  Kindly refactor this file once more

Reply:
I have refactored the code. Thank you @FrederickMih